pkg/webhook/configuration.go

```diff
@@ -635,7 +638,7 @@ func preUpgrade(openebsNamespace string) error {
 	for _, config := range webhookConfigList.Items {
 		if config.Labels[string(apis.OpenEBSVersionKey)] != version.Current() {
 			if config.Labels[string(apis.OpenEBSVersionKey)] == "" ||
-				util.IsCurrentLessThanNewVersion(config.Labels[string(apis.OpenEBSVersionKey)], "1.10.0") {
+				util.IsCurrentLessThanNewVersion(config.Labels[string(apis.OpenEBSVersionKey)], "1.12.0") {
```

Comment:
q: does this also support upgrades to custom tags like 1.12.0-arm or 1.12.0-ee ? ( cc: @shubham14bajpai )

Reply:
@kmova yes. It is handled in `IsCurrentLessThanNewVersion` function